doc/src/manual/arrays.md

```diff
@@ -722,9 +714,20 @@ array indirectly.  By putting the [`@views`](@ref) macro in front of an expressi
 block of code, any `array[...]` slice in that expression will be converted to
 create a `SubArray` view instead.
 
-`StridedVector` and `StridedMatrix` are convenient aliases defined to make it possible for Julia
-to call a wider range of BLAS and LAPACK functions by passing them either [`Array`](@ref) or
-`SubArray` objects, and thus saving inefficiencies from memory allocation and copying.
+A "strided" array is stored in memory and has its elements are laid out in regular offsets such that
```

Comment:
This should be either "and has its elements laid out" or "and its elements are laid out".

Reply:
Good catch, thank you!